### ISSUE TITLE
Show tabs across windows?

### ISSUE THREAD

**Issue by Noah-Houghton** [Rapptz/power-tabs#23]
Not sure what the limitations of the API are, but if this extension showed the tabs in a group across all open Firefox windows it would be better for users with multiple screens.

**Comment by onlycparra** [comment]
I agree, that is the only thing that it needs to be the perfect side tabs plug-in
